### PR TITLE
Enables woloxci

### PR DESCRIPTION
**Pull request by mdesanti**
## Summary

Enables Woloxci

### CODE REVIEW

File: .woloxci/config.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+steps:
+  copy_node_modules:
+    - cp -r $NODE_PATH/ ./
+  migrate_db:
+    - node_modules/.bin/sequelize db:migrate
+  lint:
+    - npm run lint
+  test:
+    - npm run test
```
Comment (mpizzagalli) [inline .woloxci/config.yml]: Just a little detail, in the case of test you can avoid the run keyword, you can call `npm test`

**Comment by mpizzagalli** [comment]
@mdesanti to make this work on jenkins do we need to config something on jenkins itself?
And we should turn off travis, right?

File: .woloxci/config.yml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  project_name: express-js-bootstrap
+
+services:
+  - postgres
+
+steps:
+  copy_node_modules:
+    - cp -r $NODE_PATH/ ./
+  migrate_db:
+    - node_modules/.bin/sequelize db:migrate
```
Comment (alanhala) [inline .woloxci/config.yml]: Who creates the DB?

Comment (mdesanti) [reply .woloxci/config.yml]: The postgres docker image comes with an already created DB

**Pull request merged**